Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -393,6 +393,7 @@
 "insert-media-image-preview-select-image-action-title" = "Select image";
 "insert-media-image-size-setting-custom" = "Custom";
 "insert-media-image-size-setting-default" = "Default";
+"insert-media-image-size-settings-px-unit-name" = "px";
```

Comment:
In css is 'px' ever not 'px' in non-EN usage?

Reply:
This is a display name here -

![Simulator Screen Shot - iPhone Xʀ - 2019-04-24 at 17 18 59](https://user-images.githubusercontent.com/9299317/56694594-13dee800-66b5-11e9-9dda-cd8843aae59b.png)

Reply:
ooh never mind i think i see how it's being used

Reply:
hehe dueling comments 😂